pkg/server/handlers.go

```diff
@@ -196,23 +262,24 @@ func createDeployment(w http.ResponseWriter, r *http.Request) {
 		tempPTS.Annotations = make(map[string]string)
 	}
 
-	if tempJSON.PrivateHosts != nil {
-		tempPTS.Annotations["privateHosts"] = *tempJSON.PrivateHosts
-	}
-
-	if tempJSON.PublicHosts != nil {
-		tempPTS.Annotations["publicHosts"] = *tempJSON.PublicHosts
-	}
+	tempPTS.Annotations["edge/paths"] = composePathsJSON(tempJSON.Paths)
```

Comment:
`30x/dispatcher` makes this and a few other annotation/label names configurable.  I don't think it makes sense for us to make them configurable here but we might need to in the future when we get to the point where we're using `enrober` to deploy interceptors/proxies.